behat/symfony2-extension/2.1/behat.yaml.dist

```diff
@@ -0,0 +1,7 @@
+default:
+    extensions:
+        Behat\Symfony2Extension:
+            kernel:
+                bootstrap: vendor/autoload.php
```

Comment:
another option: make it point to a file containing *only* the Dotenv loading 😄 (but a reusable extension for Behat would be cleaner)